### PR TITLE
Add Apple Mail, fix opening and macgpg description

### PR DESCRIPTION
**Pull request by dngray**

### CODE REVIEW

**Comment by jonaharagon** [comment]
GPG Mail from GPGTools is not free, it requires a ~$24 (roughly yearly) [support plan](https://gpgtools.org/support-plan) after a 30-day free trial.

**Comment by dngray** [comment]
> GPG Mail from GPGTools is not free, it requires a ~$24 (roughly yearly) [support plan](https://gpgtools.org/support-plan) after a 30-day free trial. Don't know if this makes a difference for the recommendation. It _is_ open-source with [their own license](https://github.com/GPGTools/GPGMail/blob/dev/LICENSE.txt) which is very permissive at a glance.

Probably should mention that, but no we'd still list it, as it's the only implementation that does support Apple Mail, and integrates nicely.

File: docs/email-clients.md [hunk 1]
```diff
@@ -28,6 +28,19 @@ Discover free, open-source, and secure email clients, along with some email alte
...
+### Apple Mail
+
+!!! recommendation
+
+    ![Apple Mail logo](/assets/img/email-clients/applemail.png){ align=right }
+
+    **Apple Mail** included in macOS and can be extended to have OpenPGP support with [MacGPG](/encryption/#macgpg).
+
+    !!! note
+        For iOS devices we suggest [Canary Mail](/email-clients/#canary-mail) as it has PGP support which means you can sen
```
Comment (lx-is) [inline docs/email-clients.md:40]: This sentence isn't finished

File: docs/email-clients.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Email Clients
 icon: material/email-open
 ---
-Discover free, open-source, and secure email clients, along with some email alternatives you may not have considered.
+Our recommended email clients list contains email clients that support both [OpenPGP](/encryption/#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](/multi-factor-authentication) and prevent account theft.
```
Comment (lx-is) [inline docs/email-clients.md:5]: This sounded a bit convoluted

```suggestion
Our recommendation list contains email clients that support both [OpenPGP](/encryption/#openpgp) and strong authentication such as [Open Authorization (OAuth)](https://en.wikipedia.org/wiki/OAuth). OAuth allows you to use [Multi-Factor Authentication](/multi-factor-authentication) and prevent account theft.
```

File: docs/email-clients.md [hunk 3]
```diff
@@ -28,6 +28,19 @@ Discover free, open-source, and secure email clients, along with some email alte
...
+!!! recommendation
+
+    ![Apple Mail logo](/assets/img/email-clients/applemail.png){ align=right }
+
+    **Apple Mail** included in macOS and can be extended to have OpenPGP support with [MacGPG](/encryption/#macgpg).
+
+    !!! note
+        For iOS devices we suggest [Canary Mail](/email-clients/#canary-mail) as it has PGP support which means you can sen
+
+    [Visit apple.com](https://support.apple.com/guide/mail/welcome/mac){ .md-button .md-button--primary }
```
Comment (lx-is) [inline docs/email-clients.md:42]: Use the privacy policy linked on the Apple Mail App Store page

```suggestion
    [Visit apple.com](https://support.apple.com/guide/mail/welcome/mac){ .md-button .md-button--primary } [Privacy Policy](https://www.apple.com/legal/privacy/en-ww/){ .md-button }
```

**Review by lx-is — CHANGES_REQUESTED** [review]

File: docs/email-clients.md (hunk 1 above)
Comment (dngray) [reply docs/email-clients.md]: Oh, yeah must have got distracted.

**Comment by lx-is** [comment]
> GPG Mail from GPGTools is not free

There is [Free-GPGMail](https://github.com/Free-GPGMail/Free-GPGMail) which removes the DRM. "While the seller of the commercial product has the right to charge for a support plan and place DRM into their product, it is still BSD and GPL licensed software which requires the source code to be available."

**Pull request merged**